### PR TITLE
DOCSP-33295 Allow user to Run Aggregate

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
## DESCRIPTION
Document option to direct run an aggregation using a Search Index. Also added documentation to edit search and vector search indexes and run vector search queries.

## STAGING
https://preview-mongodbkanchanamongodb.gatsbyjs.io/compass/DOCSP-33295/query/atlas-search/
https://preview-mongodbkanchanamongodb.gatsbyjs.io/compass/DOCSP-33295/indexes/create-search-index/

## JIRA
https://jira.mongodb.org/browse/DOCSP-33295

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=65ce62a67111165982ffa6fe

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/query/atlas-search.txt [hunk 1]
```diff
@@ -0,0 +1,46 @@
+.. _run-atlas-search-query:
+
+========================
+Run Atlas Search Queries
+========================
+
+.. default-domain:: mongodb
+
+For data on the Atlas cluster, if you :ref:`created the Atlas Search index
+<compass-create-search-index>`, you can run Atlas Search
```
Comment (ltran-mdb2) [inline source/query/atlas-search.txt:10]: [suggestion] I would suggest putting the [conditional clause first ](https://www.mongodb.com/docs/meta/style-guide/writing/write-clear-concise-sentences-paragraphs/#use-a-consistent-sentence-structure) re the style guide

File: source/query/atlas-search.txt [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+:pipeline:`$search` and :pipeline:`$searchMeta` queries against the
+indexed data from |compass|. If you have an existing Atlas Vector Search 
+index on your data in Atlas, you can run :pipeline:`$vectorSearch`
+queries against the indexed data from |compass|. 
+
+Procedure
+---------
```
Comment (ltran-mdb2) [inline source/query/atlas-search.txt:17]: Can we make this "Steps" like in the task prototype page?

**Review by ltran-mdb2 — COMMENTED** [review]
Thanks for picking this work up @kanchana-mongodb ! I left a couple comments for consideration

File: source/indexes/create-search-index.txt [hunk 3]
```diff
@@ -31,8 +31,11 @@ To create an Atlas Search index, your deployment must be either:
 
 Additionally, your deployment must run MongoDB version 7.0 or later.
 
-Steps
------
+Procedures
```
Comment (ltran-mdb2) [inline source/indexes/create-search-index.txt:34]: I think we could probably keep this one as Steps to align with the task page prototype

**Review by ltran-mdb2 — APPROVED** [review]
LGTM with one small suggestion! Thanks @kanchana-mongodb !

File: source/indexes/create-search-index.txt (hunk 3 above)
Comment (kanchana-mongodb) [reply source/indexes/create-search-index.txt:34]: These are not sequential steps, but separate/disparate (create, modify, remove) actions that the user can take on the index. I think Procedures is correct in this context.

**Review by mabaasit — APPROVED** [review]

**Pull request merged**